src/core/client/admin/components/StoryInfoDrawer/StorySettingsContainer.css

```diff
@@ -0,0 +1,33 @@
+.tabBar {
+  margin-top: var(--spacing-3);
+}
+
+.tab {
+}
+
+.activeTab {
+  border-bottom: 1px solid var(--color-teal-500);
+}
+
+.tabIcon {
+  margin-right: var(--spacing-1);
+}
+
+.configureStory {
+  padding-top: var(--spacing-1);
+  font-weight: var(--font-weight-secondary-regular);
```

Comment:
If we're using `font-weight-secondary-regular` be careful that you also set `font-family: var(--font-family-secondary);` as well. The `secondary` font vars are made to work with the `font-family-secondary`.

It can feel a bit like pulling variables out of thin air, but if you want to see the vars they're defined in `src/core/client/ui/theme/sharedVariables.ts`.